### PR TITLE
HTML/CSS & JavaScript capstone project - Hackathon page

### PR DESCRIPTION
**Pull request by DhuruvSingh**
## **Added More pages and features**

1. Added a home page
2. Added an about page
3. Set up a program page
4. Added the scripts of the three pages
5. Set up the dynamic data
6. Tested the webpages

### Loom Link: [Click Here](https://www.loom.com/share/e5ce326c95594c7fa531c8426fa6219f)

> Please Review and Give your valuable feedback

### CODE REVIEW

File: css/about.css [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+  padding-top: 4rem;
+}
+
+.showMenu {
+  transform: translateY(0);
+}
+
+.material-icons {
+  font-size: 60px;
+}
```
Comment (EricMbouwe) [inline css/about.css:454]: - [ ] There is a bug on your mobile menu hamburger icon and close icon which are both shown when we switch on mobile.
ham-menu

The reason for this is that somewhere in your CSS code both are initialized to be displayed `display: inline-block;` meanwhile, we only want the hamburger menu to be displayed first and the close icon to be hidden.

css-ham

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @DhuruvSingh, 

Good job so far! 🔥
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

# Suggested changes ♻
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by DhuruvSingh** [comment]
### Changes

- Fixed Hamburger close bug

> hope you are satisfied @EricMbouwe

**Review by keroloussamy — APPROVED** [review]
Hi @DhuruvSingh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DhuruvSingh** [comment]
# Thank you so much @EricMbouwe & @EricMbouwe for your valuable feedback.

**Pull request merged**